themes/default/content/docs/guides/self-hosted/ecs-hosted.md

```diff
@@ -0,0 +1,79 @@
+---
+title: ECS-Hosted Install
+menu:
+    userguides:
+        parent: self_hosted
+        identifier: self_hosted_ecs_hosted_install
+        weight: 20
+meta_desc: Installer for deploying the self-hosted Pulumi service in ECS.
+---
+
+The [ECS-Hosted Installer](https://github.com/pulumi/pulumi-self-hosted-installers/ecs-hosted) is used to deploy the self-hosted Pulumi service in Amazon Elastic Container Service.  
+
+## Prerequisites
+
+The customer is required to provide and manage the following:
+
+* AWS VPC with
+  * At least 2 public subnets available.
+  * At least 2 private subnets available.
+  * At least 2 isolated subnets available.
+    * An "isolated" subnet means it can only route traffic within the subnet. So there is no NAT gateway.
+* Route53 hosted zone.
+* ACM Certiciate that covers FQDNs of the form (note `{subdomain}` is optional):
+  * `{subdomain}.{zoneDomainName}`
+  * `api.{subdomain}.{zoneDomainName}`
+  * `app.{subdomain}.{zoneDomainName}`
+* KMS key to be used the self-hosted Pulumi Service for encryption/decryption purposes.
+
+## ECS-Hosted Deployment
+
+The ECS-hosted installation of Pulumi deploys the following services:
+
+* ECS - Managed ECS Cluster
+* Fargate - Managed Container Service
+* RDS Aurora - Managed MySQL DB for persistent state and automated replication and snapshotting.
+* S3 - Object storage for checkpoints and policy packs.
+* CloudWatch Logs - Centralized logging for all cluster pods.
+* Route53 - Managed DNS records.
+* NLB - Managed L4/application traffic and TLS termination.
+* ACM - Managed Public TLS certificates.
+
+### Pulumi deploying Pulumi
+
+This installer uses Pulumi to deploy the Pulumi service.  
+In this case, one uses the pulumi CLI with a self-managed backend (e.g. S3) to deploy all services listed above to stand up the self-hosted Pulumi Service.  
+The installation package includes Pulumi project code so that you can deploy the service by running `pulumi up`.
```

Comment:
Same changes here as well:

```suggestion
This installer uses Pulumi to deploy the Pulumi Service.  In this case, one uses the Pulumi CLI with a self-managed backend (e.g. S3) to deploy all services listed above to stand up the self-hosted Pulumi Service. The installation package includes Pulumi project code so that you can deploy the Service by running `pulumi up`.
```

Reply:
pushing an update to address this.